src/main/java/Action.java

```diff
@@ -0,0 +1,150 @@
+
+                    if (nextRoom == null) {
+                        System.out.println(Main.ANSI_BLUE + "\nThe room name is invalid. Please ask for help.\n" + Main.ANSI_RESET);
+                        break;
+                    } else {
+                        player.setCurrentRoom(nextRoom);
+                        System.out.println(Main.ANSI_BLUE + "\nYou have moved to " + nextRoom.getRoomName() + ".\n" + player.getCurrentRoom().getScript() + "\n" + Main.ANSI_RESET);
+                        break;
+                    }
+                }
+                else {
+                    System.out.println(Main.ANSI_BLUE + "\nDid you mean 'move to'?\n" + Main.ANSI_RESET);
+                    break;
+                }
+            case "pick":
+                if (command[1].equals("up")) {
+                    Room currentRoom = player.getCurrentRoom();
+
+                    if (command.length < 3) {
+                        System.out.println(Main.ANSI_BLUE + "\nYou cannot pick up nothing.\n" + Main.ANSI_RESET);
+                        break;
+                    }
+
+                    String itemName = command[2];
+                    Item item = currentRoom.getItemFromName(itemName);
+                    if (item == null) {
+                        System.out.println(Main.ANSI_BLUE + "\nThe item name is invalid. Please ask for help.\n" + Main.ANSI_RESET);
+                        break;
+                    } else {
+                        player.addToInventory(item);
+                        currentRoom.removeItemFromRoom(itemName);
+                        System.out.println(Main.ANSI_BLUE + "\nYou have picked up " + itemName + "." + Main.ANSI_RESET);
+                        System.out.println(Main.ANSI_BLUE + printInventory(player.getInventory())
+                                + ".\n" + Main.ANSI_RESET);
+                        break;
+                    }
+
+                } else {
+                    System.out.println(Main.ANSI_BLUE + "\nDid you mean 'pick up'?\n" + Main.ANSI_RESET);
+                    break;
+                }
+            case "inspect":
+                if (command.length == 2) {
+                    String itemName = command[1];
+
+                    Item item = Main.itemMap.get(itemName);
+
+                    if (item == null) {
+                        System.out.println(Main.ANSI_BLUE + "\nThat item does not exist.\n" + Main.ANSI_RESET);
+                        break;
+                    } else {
+                        System.out.println(Main.ANSI_BLUE + "\n" + item.getUsage() + "\n" + Main.ANSI_RESET);
+                        break;
+                    }
+
+                } else {
+                    System.out.println(Main.ANSI_BLUE + "\nDid you mean 'inspect item'?\n" + Main.ANSI_RESET);
+                    break;
+                }
+            case "help":
+                ArrayList<Item> playerItems = player.getInventory();
+                String playerInventory = "";
```

Comment:
Might be blind but I don't see what this variable playerInventory is for

Reply:
^ Once the variable is created it never appears to be used.